backend/inventories/views.py

```diff
@@ -0,0 +1,38 @@
+from django.shortcuts import render
+from rest_framework import generics, permissions, status, viewsets
+from rest_framework.response import Response
+from rest_framework.exceptions import NotFound
+
+from .models import InventoryReport, InventorySupply
+from .serializers import InventoryReportSerializer, InventorySupplySerializer
+
+
+
+class InventoryReportCreateView(generics.CreateAPIView):
+    permission_classes = (permissions.IsAdminUser,)
+    queryset = InventoryReport.objects.all()
+    serializer_class = InventoryReportSerializer
+
+
+class InventoryReportListView(generics.ListAPIView):
+    permission_classes = (permissions.IsAuthenticated,)
+    queryset = InventoryReport.objects.all()
+    serializer_class = InventoryReportSerializer
```

Comment:
Listing stocktakings should not return theier contents, only theier ids and names

Reply:
Done. I've provided separate serializer for this purpose, which besides of id, date and name, also returns number of total supplies assigned to each report and number of supplies checked out already.